buildconfig/stubs/pygame/__init__.pyi

```diff
@@ -51,6 +51,11 @@ from .math import Vector2 as Vector2, Vector3 as Vector3
 from .cursors import Cursor as Cursor
 from .bufferproxy import BufferProxy as BufferProxy
 from .mask import Mask as Mask
+from .event import Event as Event
```

Comment:
As mentioned in the comment at the top of the file, this is an autogenerated file. Please edit `buildconfig/stubs/gen_stubs.py` instead